demo-config1.yml

```diff
@@ -0,0 +1,13 @@
+# working with Pac4jFactory1:
+pac4j:
+  configClass: org.pac4j.demo.dw.factory.DemoConfigFactory1
```

Comment:
I don't see a callbackUrl property that are in demo-config.yml and demo-config2.yml. Is that field optional and what does that mean comparing to callbackUrl property under configProperties?

Can we add some comments in the yml files for ignorant people such as me :) ?